AmplifyPlugins/Core/AWSPluginsCore/Auth/Configuration/AWSAuthorizationConfiguration.swift

```diff
@@ -45,6 +46,15 @@ extension AWSAuthorizationConfiguration {
             return .apiKey(config)
     }
 
+    private static func awsLambdaAuthConfiguration(region: AWSRegionType?)
```

Comment:
Can we remove this region so customers don't have to set it?

Reply:
Not needed, removed